.github/actions/execute-assert-arc-e2e/action.yaml

```diff
@@ -0,0 +1,151 @@
+name: 'Execute and Assert ARC E2E Test Action'
+description: 'Queue E2E test workflow and assert workflow run result to be succeed'
+
+inputs:
+  github-auth-token:
+    description: 'GitHub access token to queue workflow run'
+    required: true
+  github-repo-owner:
+    description: "The repository owner name that has the test workflow file, ex: actions"
+    required: true
+  github-repo-name:
+    description: "The repository name that has the test workflow file, ex: test"
+    required: true
+  github-workflow-file:
```

Comment:
nit: I'm not really sure why we need `github` as a prefix to any of these

Reply:
I copied from the setup-arc-e2e action. 😄 
I can remove it if you want.

Reply:
I'll leave it up to you, it's not adding any value to be honest

Reply:
will remove them.